openupgradelib/openupgrade.py

```diff
@@ -960,19 +962,34 @@ def reactivate_workflow_transitions(cr, transition_conditions):
             (condition, transition_id))
 
 
-def migrate(no_version=False):
+def migrate(no_version=False, v9env=False, context=None, *args, **kwargs):
```

Comment:
How about defaulting to `v9env=None`? Then, if it is True or (None and version > 9), enable it. So, you allow old-style lovers to pass `v9env=False`.